src/components/Header.tsx

```diff
@@ -0,0 +1,123 @@
+import { Dispatch, FC, FormEvent, useEffect, useState } from 'react';
+import { USER_ID, addTodo, updateTodos } from '../api/todos';
+
+import useFocusInput from '../hooks/useFocusInput';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  onErrorMessage: (message: string) => void;
+  setTodos: Dispatch<React.SetStateAction<Todo[]>>;
```

Comment:
`setTodos` Should not be here. please remove/replace it.

As long as Header is responsible only for adding single `todo` it's props should reflect this.
`setTodos` allows for instance removing all todos - that is way beyond the Header responsibility.